examples/uptime.go

```diff
@@ -8,14 +8,14 @@ import (
 	"github.com/cloudfoundry/gosigar"
```

Comment:
Curious why an import alias (`sigar`) as is used in elsewhere[1] wasn't added / needed here?

Possibly an un-aliased import should be used in all cases?

[1] 
- https://github.com/cloudfoundry/gosigar/pull/64/files#diff-16369e027e97504df424187da2a23c1a384e925dd3593d1dcab5b92631681b1c
- https://github.com/cloudfoundry/gosigar/pull/64/files#diff-ecb176b62790863d682cf735ef8eafd0862ad134b85454e777ce5085f8161c35
- https://github.com/cloudfoundry/gosigar/pull/64/files#diff-46492863d631aeead67062fc8f7cc097d0d66a5609b218928da58f88a1787b6f

Reply:
This was done by [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports). I've now used the same alias in all files.